### PR TITLE
UI: add Fees to Orders

### LINKED ISSUE #3084: Order Details: Rename Labels and Add Fees
This is for the Orders → Order Details page.

## Order Total Updates

- “Paid by Customer” is becoming “Paid” for simplicity (for all orders, refunded or not)
- When a refund is processed, “Net Total” is becoming “Net Payment” to better reflect that it’s showing the total amount paid after the refund.

## Fees

You can add a fee item when creating a new product in wp-admin. The fee currently does not show up in Order Details. 

Add a Fee row in Order Details. 

Ref p91TBi-36J-p2.

### PR DESCRIPTION
**Pull request by ctarda**
Closes #3084 

This is the fourth and final step to add support for Fees to the Order Details page. (#3084).

* Yosemite: #3466 
* Storage: #3461
* Networking: #3452

This PR adds support for Fees in the UI.

| | Light Appearance | Dark Appearance |
| ---- | ---- | ---- |
| With fees  |   |  |
| Without fees  |   |  |

## Changes
* Expose total fees in `OrderPaymentDetailsViewModel`
* Update `LedgerTableViewCell` to add the Fees line
* Move the logic deciding visibility of taxes and coupons to OrderPaymentDetailsViewModel
* Add unit tests to cover the cell and the view model (renaming some of the pre-existing tests to match the naming of other test suites)

## How to test
* Add fees to one or more than one orders in wp-admin. For extra credit, add a single Fee to order A, and more than one Fee to order B
* Checkout the branch, navigate to an order without Fees. The Fees line should not be visible
* Navigate to order A (one Fee) and B (more than one Fee). Notice if the Fees are visible, and it the amount matches the fees assigned in wp-admin. 

Update release notes:
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: WooCommerce/WooCommerceTests/ViewRelated/OrderPaymentDetailsViewModelTests.swift [hunk 1]
```diff
@@ -23,9 +25,14 @@ final class OrderPaymentDetailsViewModelTests: XCTestCase {
 
         anotherBrokenOrder = MockOrders().unpaidOrder()
         anotherBrokenOrderViewModel = OrderPaymentDetailsViewModel(order: anotherBrokenOrder)
+
+        orderWithFees = MockOrders().orderWithFees()
+        orderWithFeesViewModel = OrderPaymentDetailsViewModel(order: orderWithFees)
```
Comment (jaclync) [inline WooCommerce/WooCommerceTests/ViewRelated/OrderPaymentDetailsViewModelTests.swift]: nit: I think we might want to DI `CurrencySettings` to match the settings used for the fees value? If the previously launched store has different currency settings from `CurrencySettings()` the tests might not pass

```suggestion
        orderWithFeesViewModel = OrderPaymentDetailsViewModel(order: orderWithFees, currencySettings: CurrencySettings())
```

**Review by jaclync — APPROVED** [review]
Thanks for working on this feature! Code looks good, just two minor things I noticed:

- The issue #3084 also mentions updating "Paid by Customer" to "Paid" - maybe we can update this string in this PR or a separate one?
- The fees labels seem to have smaller font than other labels (screenshot below). Maybe we can apply the same style as other labels (body style) in `configureLabels`?

https://github.com/woocommerce/woocommerce-ios/blob/1af8031004a3384802303effa73722ea966636b4/WooCommerce/Classes/ViewRelated/ReusableViews/LedgerTableViewCell.swift#L164-L175

File: WooCommerce/WooCommerceTests/ViewRelated/OrderPaymentDetailsViewModelTests.swift (hunk 1 above)
Comment (ctarda) [reply WooCommerce/WooCommerceTests/ViewRelated/OrderPaymentDetailsViewModelTests.swift]: Good catch, thanks

**Comment by ctarda** [comment]
Thanks for the review folks. I pushed an update to incorporate @jaclync 's feedback, and will merge when CI is ✅

**Comment by astralbodies** [comment]
Yay! Thanks for doing this work, @ctarda!

**Pull request merged**